runtime/test/manifest-test.js

```diff
@@ -1218,6 +1218,42 @@ resource SomeName
     assert(validRecipe.isResolved());
   });
 
+  it('can process a schema alias', async () => {
+    let manifest = await Manifest.parse(`
+      alias schema This That as SchemaAlias
+      alias schema * extends SchemaAlias as Extended
+    `);
+    assert.isNotNull(manifest.findSchemaByName('SchemaAlias'));
+    assert.sameMembers(manifest.findSchemaByName('Extended').names, ['This', 'That']);
+  });
+
+  it('expands schema aliases', async () => {
+    let manifest = await Manifest.parse(`
+      alias schema Name1 as Thing1
+        Text field1
+      alias schema Name2 as Thing2
+        Text field2
+      particle P in 'p.js'
+        P(in Thing1 Thing2 {Text field1, Text field3} param)
+    `);
+  });
+
+  it('fails when expanding conflicting schema aliases', async () => {
+    try {
+      let manifest = await Manifest.parse(`
+        alias schema Name1 as Thing1
+          Text field1
+        alias schema Name2 as Thing2
+          Number field1
+        particle P in 'p.js'
+          P(in Thing1 Thing2 {} param)
+      `);
+      assert.fail();
+    } catch (e) {
+      assert.include(e.message, 'Could not merge schema aliases');
+    }
+  });
+
   it('can relate inline schemas to generic connections', async () => {
```

Comment:
maybe check that something like
```
alias schema Name 1 as Thing1
  Text field1
particle P in 'p.js'
  P(in Thing1 { Number field1})
```

fails too?

Reply:
Done!